Applications/Games/Subnautica/Steam/script.js

```diff
@@ -12,29 +13,33 @@ new SteamScript()
     .author("Zemogiter")
     .applicationHomepage("https://unknownworlds.com/subnautica/")
     .wineDistribution("upstream")
-    .wineVersion(getLatestStableVersion)
+    .wineVersion(getLatestDevelopmentVersion)
     .wineArchitecture("amd64")
     .appId(264710)
     .preInstall((wine) => {
-        const wizard = wine.wizard();
-
-        wizard.message(
-            tr("You can make the game smoother by using this: https://github.com/lutris/lutris/wiki/How-to:-Esync")
-        );
-
         new Vcrun2013(wine).go();
         new Corefonts(wine).go();
         new DXVK(wine).go();
-
+        new OverrideDLL(wine)
+            .withMode("disabled", ["nvapi", "nvapi64"])
+            .go();
         new VirtualDesktop(wine).go();
+        const dxvkConfigFile = wine.prefixDirectory() + "/drive_c/dxvk.conf";
+        touch(dxvkConfigFile);
+        writeToFile(dxvkConfigFile, "dxgi.nvapiHack = False");
     })
     .postInstall((wine) => {
         const wizard = wine.wizard();
 
         wizard.message(
             tr(
-                "Due to a potential conflict with Vulkan, shader mods break the game (the executable file works but no window is displayed)."
+                "Due to a potential conflict with Vulkan, shader mods may break the game (the executable file works but no window is displayed)."
             )
         );
     })
-    .gameOverlay(false);
+    .gameOverlay(false)
+    .environment((wine) => {
+        const dxvkConfigFile = wine.prefixDirectory() + "/drive_c/dxvk.conf";
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4800368)